_infra/helm/securebanking-openbanking-uk-rcs/templates/deployment.yaml

```diff
@@ -14,6 +14,18 @@ spec:
       maxSurge: {{ .Values.deployment.rollingUpdate.maxSurge }}
       maxUnavailable: {{ .Values.deployment.rollingUpdate.maxUnavailable }}
     {{ end }}
+  {{ if eq .Values.deployment.useAffinity "enabled" }}
+  affinity:
+    podAntiAffinity:
```

Comment:
Why antiAffinity?

Seems quite a specify set of rules to opt into, would it be better to allow a whole block of yaml to configure affinity to be dropped in?